tensorflow/lite/experimental/litert/vendors/qualcomm/core/utils/miscs.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Qualcomm Innovation Center, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "tensorflow/lite/experimental/litert/vendors/qualcomm/core/utils/miscs.h"
+
+#include "absl/types/span.h"
+
+namespace qnn {
+void ConvertDataFromInt16toUInt16(absl::Span<const std::int16_t>& src,
```

Comment:
Pass by value, span is already a lightweight container.
```suggestion
void ConvertDataFromInt16toUInt16(absl::Span<const std::int16_t> src,
```